packages/components/src/List/ListComposition/VList/VList.component.js

```diff
@@ -8,7 +8,8 @@ function VList(props) {
 	return <VirtualizedList collection={collection} type={displayMode.toUpperCase()} {...props} />;
 }
 
-VList.Content = VirtualizedList.Content;
-VList.propTypes = VirtualizedList.propTypes;
-
+Object.entries(VirtualizedList).forEach(([key, value]) => {
```

Comment:
Could add a little comment here plz ?
I didn't catch directly that it is used to avoid the verbosity of all the column references created in this pr.